chainer/training/updater.py

```diff
@@ -307,6 +308,10 @@ def update_core(self):
                 in_vars = variable.Variable(in_arrays)
                 losses.append(loss_func(in_vars))
 
+        # For _uninitialized_params
```

Comment:
Why `clear_grads` is set before call loss_func?
https://github.com/pfnet/chainer/pull/1551/files#diff-32a6fe336029b64713fe8980fe17f085R293